src/ui/components/results/resultsheadercomponent.js

```diff
@@ -44,6 +44,12 @@ export default class ResultsHeaderComponent extends Component {
      */
     this.resultsLength = data.resultsLength || 0;
 
+    /**
+     * The compiled custom results count template, if the user specifies one.
+     * @type {Function}
+     */
+    this._compiledResultsCountTemplate = this._renderer.compile(this._config.resultsCountTemplate);
```

Comment:
If no resultsCountTemplate is provided, does this compile to undefined or ''?

Reply:
when resultsCountTemplate is not provided, resultsCountTemplate will default to ""
in that case, this compiled template will be a function that returns ""